controllers/build/modulehelper.go

```diff
@@ -41,3 +44,26 @@ func (moduleHelper) ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides .
 
 	return args
 }
+
+func (m moduleHelper) GetReleventBuild(mod ootov1alpha1.Module, km ootov1alpha1.KernelMapping) *ootov1alpha1.Build {
```

Comment:
This method doesn't use the `moduleHelper` obj. Should we make it a function instead of a method?

Reply:
I always prefer using methods instead of module functions. Makes code clear and, in addition, makes it easier to write unit tests